source/specifications/schemas/pylock.schema.json

```
@@ -0,0 +1,345 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://json.schemastore.org/pylock.json",
```

```suggestion
  "$id": "https://packaging.python.org/en/latest/specifications/schemas/pylock.schema.json",
```

Now that we have a linkable copy here, we should use it as the ID rather than the schemastore version.

---

Ah, yup!